src/components/sections/Section/Section.scss

```diff
@@ -0,0 +1,9 @@
+.pages__section {
```

Comment:
Not sure. This is because the sections should be self contained and not use any global CSS. The way sections are being used today is only a wrapper to `src/components/ui` components, which is not ideal. I'll have another PR fixing this issue and making sections actually self-contained sections. 

The goal of sections is to integrate nicely with the CMS. The idea is that a section can be removed/added and their assets are also removed added from/to the page. That's why sections should be self contained.

Reply:
Ooh, I see! That's fine by me then.